frontend/src/metabase/collections/components/CollectionContent/CollectionItemsTable.tsx

```diff
@@ -157,6 +159,9 @@ export const CollectionItemsTable = ({
           !loading && !hasPinnedItems && unpinnedItems.length === 0;
 
         if (isEmpty && !loadingUnpinnedItems) {
+          if (isEmbeddingSdk && EmptyContentComponent) {
+            return <EmptyContentComponent />;
+          }
```

Comment:
Wondering if we can use `EmptyContentComponent` and not having to rely on a `isEmbeddingSdk`-specific check.

(we could even create a default EmptyContentComponent for the main Metabase app)

Reply:
That sounds nice!